lunatrace/bsl/backend/src/fixtures/large-repo/package.json

```diff
@@ -0,0 +1,122 @@
+{
+  "author": "",
```

Comment:
this was missing the lockfile because the gitignore ignored it, causing tests to fail.